### ISSUE TITLE
[MSFT-209] Sample challenge page: The 'arrow' which when clicked takes the user back to the challenge feed page is being read as 'ico-arrow-big-left' by screen readers.

### ISSUE THREAD

**Issue by veshu** [importtest/importest#68]
**Steps to Reproduce**
1. Go to the sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure the screen reader is switched on. 3. Press the tab key from the 'find members by username or skill' textbox. 4. Note the screen reader reads 'link graphic ico-arrow-big-left'. 

**Expected Result**
The label for this link should be 'back' indicating its purpose.

**Actual Result**
The label being read by the screen reader for the back link is inappropriate.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.4 Link Purpose (In Context)
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2036
